src/main/kotlin/org/elm/lang/core/psi/elements/ElmGlslCodeExpr.kt

```diff
@@ -1,8 +1,27 @@
 package org.elm.lang.core.psi.elements
 
 import com.intellij.lang.ASTNode
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiLanguageInjectionHost
+import org.elm.ide.injection.GlslEscaper
 import org.elm.lang.core.psi.ElmAtomTag
 import org.elm.lang.core.psi.ElmPsiElementImpl
+import org.elm.lang.core.psi.ElmPsiFactory
+import org.elm.lang.core.psi.ElmTypes
+
+
+class ElmGlslCodeExpr(node: ASTNode) : ElmPsiElementImpl(node), ElmAtomTag, PsiLanguageInjectionHost {
+    val content: List<PsiElement> get() = findChildrenByType(ElmTypes.GLSL_CODE_CONTENT)
+
+    override fun updateText(text: String): PsiLanguageInjectionHost {
+        val expr = ElmPsiFactory(project).createGlslExpr(text)
```

Comment:
I'm curious if there's a more efficient way to do this. given the way we lex these blocks, I couldn't think of one.

Reply:
No idea. This seems fine for now.